src/main/resources/game_publishers.csv

```diff
@@ -11,7 +11,6 @@ Sierra Entertainment, 63
 Capcom, 48
 Codemasters, 47
 LucasArts, 47
-Inc., 45
```

Comment:
Tady bych se zaměřil na správné parsování dat. Pak by neměl vůbec vzniknout tento záznam.

Reply:
😆 hrála jsem si s tim 2 hodiny, plus jsem udělala nějaké handlery pro situace, kdy bude vstupní parametr prázdný/null a testy. Koukni na to, jestli sis to představoval nějak takhle. Jestli ne - mrknu na to v pondělí. ted fičím vstříc víkendu :).